openshift-with-appstudio-test/e2e/client.go

```diff
@@ -1,6 +1,3 @@
-//go:build normal || periodic
```

Comment:
@gabemontero I messed up in the previous PR. These tags were not needed. What needed an update was the [gosec and golangci-lint command args](https://github.com/redhat-appstudio/jvm-build-service/pull/207/files#diff-b3ff1ee6de56e14f44fca2bf15023e30ac069e3daaf0cf204d9c4f94712974f8)